full-service/src/config.rs

```diff
@@ -97,7 +97,7 @@ impl APIConfig {
                     signature.product_id(),
                     signature.version(),
                 );
-                mr_signer_verifier.allow_hardening_advisories(&["INTEL-SA-00334"]);
+                mr_signer_verifier.allow_hardening_advisories(&["INTEL-SA-00334", "INTEL-SA-00615"]);
```

Comment:
Ideally this would use `mc_fog_ingest_enclave_measurement::HARDENING_ADVISORIES`, but `mc_consensus_enclave_measurement::HARDENING_ADVISORIES` should be equivalent.